model-output/README.md

```diff
@@ -1,3 +1,331 @@
+
+### `target_end_date`
+
+Values in the `target_end_date` column must be a date in the format
+
+    YYYY-MM-DD
+    
+This is the last date of the forecast target's week. This will be the date of the Saturday at the end of the forecasted week. As a reminder, the `target_end_date` is the end date of the week during which the admissions are reported (see the data processing section for more details). Within each row of the submission file, the `target_end_date` should be equal to the `reference_date` + `horizon`*(7 days).
+
+
+### `location`
+
+Values in the `location` column must be one of the "locations" in
+this [location information file](../auxiliary-data/locations.csv) which
+includes numeric FIPS codes for U.S. states,  territories, and districts, as well as "US" for national forecasts, included in the FluSight forecasting collaboration.
+
+Please note that when writing FIPS codes, they should be written in as a character string to preserve any leading zeroes.
+
+### `output_type`
+
+Values in the `output_type` column are either
+
+-   "quantile" or
+-   "pmf".
+
+This value indicates whether that row corresponds to a quantile forecast for incident hospital admissions or the probability mass function (pmf) of a categorical forecast for rate-trend. Quantile forecasts are used in visualizations and to construct the primary FluSight ensemble.
+
+### `output_type_id`
+Values in the `output_type_id` column specify identifying information for the output type.
+
+#### quantile output
+
+When the predictions are quantiles, values in the `output_type_id` column are a quantile probability level in the format
+
+    0.###
+
+ This value indicates the quantile probability level for for the
+`value` in this row.
+
+Teams must provide the following 23 quantiles:
+
+0.01, 0.025, 0.05, 0.1, 0.15, 0.2, 0.25, 0.3, 0.35, 0.4,
+0.45, 0.5, 0.55, 0.6, 0.65, 0.7, 0.75, 0.8, 0.85, 0.9,
+0.95, 0.975, and 0.99
+
+R: c(0.01, 0.025, seq(0.05, 0.95, by = 0.05), 0.975, 0.99) 
+Python: quantiles =
+np.append(np.append([0.01,0.025],np.arange(0.05,0.95+0.05,0.05)),
+[0.975,0.99])
+
+
+### `value`
+
+Values in the `value` column are non-negative numbers indicating the "quantile" or "pmf" prediction for this row. For a "quantile" prediction, `value` is the inverse of the cumulative distribution function (CDF) for the target, location, and quantile associated with that row. For example, the 2.5 and 97.5 quantiles for a given target and location should capture 95% of the predicted values and correspond to the central 95% Prediction Interval. 
+
+### Example tables

```

Comment:
maybe this level of FLuSight-specific detail does not need to be included here in this bare-bones template?